contrib/platform/src/com/sun/jna/platform/win32/WinNT.java

```diff
@@ -2905,6 +2905,114 @@ public SYSTEM_LOGICAL_PROCESSOR_INFORMATION(Pointer memory) {
         int RelationAll = 0xFFFF;
     }
 
+    /**
+     * Contains information about the relationships of logical processors and related hardware. The
+     * GetLogicalProcessorInformationEx function uses this structure.
+     */
+    class SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX extends Structure {
+        public static final List<String> FIELDS = createFieldsOrder("relationship", "size", "processor");
+
+        /**
+         * The type of relationship between the logical processors.
+         */
+        public WinDef.DWORD relationship;
+
+        /**
+         * The size of the structure.
+         */
+        public WinDef.DWORD size;
+
+        /**
+         * A PROCESSOR_RELATIONSHIP structure that describes processor affinity. This structure contains valid data only if
+         * the Relationship member is RelationProcessorCore or RelationProcessorPackage.
+         *
+         * This is actually a union with the NUMA and other data types.
+         */
+        public PROCESSOR_RELATIONSHIP processor;
+
+        public SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX()
+        {}
```

Comment:
Don't leave this empty - better to add the call to _super()_ - even though it is implicit, as this enables placing a debug breakpoint in the constructor.